index.html

```diff
@@ -816,36 +816,56 @@
           </dt>
         </dl>
         <p>
-          Upon receiving a <a>push message</a> for a <a>push subscription</a> from the <a>push
-          service</a> the <a>user agent</a> MUST run the following steps:
+          Upon receiving a <a>push message</a> from the <a>push service</a> the <a>user agent</a>
```

Comment:
micro nit: having a comma after `push service` makes the sentence a bit easier to read.